src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -326,12 +324,6 @@ const IOURequestStepAmountWithOnyx = withOnyx<IOURequestStepAmountProps, IOURequ
             return `${ONYXKEYS.COLLECTION.SPLIT_TRANSACTION_DRAFT}${transactionID}`;
         },
     },
-    draftTransaction: {
```

Comment:
`withFullTransactionOrNotFound()` returns draftTransaction or transaction depending upon condition. So I think we can remove this